tests/Unit/NumericalAlgorithms/Spectral/Test_Mesh.cpp

```diff
@@ -242,6 +243,29 @@ void test_serialization() noexcept {
               {{Spectral::Quadrature::GaussLobatto, Spectral::Quadrature::Gauss,
                 Spectral::Quadrature::GaussLobatto}}});
 }
+
+void test_option_parsing() {
```

Comment:
Would it make sense to template this on `size_t Dim` so you can test meshes of any dim?